packages/video_player/video_player/lib/video_player.dart

```diff
@@ -5,77 +5,16 @@
 import 'dart:async';
 import 'dart:io';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/material.dart';
 import 'package:meta/meta.dart';
 
-final MethodChannel _channel = const MethodChannel('flutter.io/videoPlayer')
-  // This will clear all open videos on the platform when a full restart is
-  // performed.
-  ..invokeMethod<void>('init');
+import 'package:video_player_platform_interface/video_player_platform_interface.dart';
 
-/// Describes a discrete segment of time within a video using a [start] and
-/// [end] [Duration].
-class DurationRange {
```

Comment:
You should export `DurationRange` and `DataSourceType` from `video_player_platform_interface` so the public API of `video_player` doesn't change

Reply:
Oh, good catch. I added the export.